### PR TITLE
support for vpc connector for cloud run

### PR DESCRIPTION
**Pull request by watchinharrison**
fixes https://github.com/terraform-providers/terraform-provider-google/issues/6294

**Release Note Template for Downstream PRs (will be copied)**

```release-note:enhancement
added vpc_connector property to cloud run service resource
```

### CODE REVIEW

File: products/cloudrun/api.yaml [hunk 1]
```diff
@@ -260,6 +263,11 @@ objects:
       Is required when creating resources. Name is primarily intended
       for creation idempotence and configuration definition. Cannot be updated.
       More info: http://kubernetes.io/docs/user-guide/identifiers#names
+  - !ruby/object:Api::Type::String
+    name: vpc_connector
+    description: The VPC connector the cloud run instance will use
+    required: false
```
Comment (danawillow) [inline products/cloudrun/api.yaml:269]: This is fine, though not strictly necessary. Typically we just omit this if it's not required.

**Comment by danawillow** [comment]
@watchinharrison are you sure this field exists in the API? Based on the comment at https://github.com/terraform-providers/terraform-provider-google/issues/6294#issuecomment-626848011, I'm not convinced it does. If it does exist, can you link me to where you found it?

**Comment by watchinharrison** [comment]
> @watchinharrison are you sure this field exists in the API? Based on the comment at [terraform-providers/terraform-provider-google#6294 (comment)](https://github.com/terraform-providers/terraform-provider-google/issues/6294#issuecomment-626848011), I'm not convinced it does. If it does exist, can you link me to where you found it?

ahh maybe not, I assumed it would be a field on the api, if it's controlled through metadata annotations, this can be closed

**Comment by danawillow** [comment]
Got it, thanks! Next time it would be helpful for our team if you could make sure any tests you wrote (or related ones) pass before sending the PR out for review (https://github.com/terraform-providers/terraform-provider-google/blob/master/.github/CONTRIBUTING.md#tests). In addition to looking at the API docs, that's a great way to know whether your API call works.

Closing this out.

**Pull request closed without merging**